### ISSUE TITLE
`scss/comment-no-empty` false positive on paragraph breaks in multi-line double-slash comments

### ISSUE THREAD

**Issue by dbjorge** [stylelint-scss/stylelint-scss#606]
Comments like this:

```scss
// First paragraph of a long comment
//
// Second paragraph
.some-rule { /* ... */ }
```

...trigger a false positive from `scss/comment-no-empty` on the blank line within the comment acting as a paragraph break. Here is an example `accept` test case that can be added to `/src/rules/comment-no-empty/__tests__/index.js` which demonstrates the issue:

```js
    {
      code: "// comment\n//\n// comment",
      description: "Multiline double slash comment with paragraph break"
    },
```

```
  ● scss/comment-no-empty › accept › [ true ] › '// comment\n//\n// comment' › Multiline double slash comment with paragraph break

    expect(received).toEqual(expected) // deep equality

    - Expected  - 1
    + Received  + 9

    - Array []
    + Array [
    +   Object {
    +     "column": 1,
    +     "line": 2,
    +     "rule": "scss/comment-no-empty",
    +     "severity": "error",
    +     "text": "Unexpected empty comment (scss/comment-no-empty)",
    +   },
    + ]

      at Object.<anonymous> (node_modules/jest-preset-stylelint/getTestRule.js:61:41)
```
